### PR TITLE
Add Banner component

### PR DESCRIPTION
**Pull request by dherault**
image
Resolves ENG-202

### CODE REVIEW

**Comment by michaeljguarino** [comment]
Is this component able to be used as a modal anchored to top/bottom, or corner of a page?

**Comment by dherault** [comment]
Once the snackbar is there it will be able to hold such a component. We can also in the meanwhile fix it wherever we want.

File: src/components/RepositoryCard copy.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
+import { Div, DivProps, Flex, H2, H3, Img, P } from 'honorable'
```
Comment (floreks) [inline src/components/RepositoryCard copy.tsx:1]: Looks like an accidental copy of the original `RepositoryCard.tsx` file

Comment (dherault) [reply src/components/RepositoryCard copy.tsx:1]: Weird, thanks

**Review by michaeljguarino — APPROVED** [review]

**Pull request merged**